src/RegularizedProblems.jl

```diff
@@ -4,6 +4,9 @@ using LinearAlgebra
 using Random, Requires
 using NLPModels
 using Distributions, Noise
+using FFTW
```

Comment:
Maybe use Requires as well for these deps (see below the `__init__()` function)?